kivy/lib/osc/OSC.py

```diff
@@ -179,7 +179,7 @@ def OSCArgument(data):
 
     if isinstance(data, string_types):
         OSCstringLength = math.ceil((len(data)+1) / 4.0) * 4
-        binary = struct.pack(">%ds" % (OSCstringLength), data)
+        binary = struct.pack(">i%ds" % (OSCstringLength), bytes(data, 'utf-8')
```

Comment:
Why do you add an int to the struct? Note that `OSCstringLength` is not being packed, it's just modifying the format string to set the data string length.